sccameraview/src/main/java/com/example/sccameraview/Camera2View.java

```diff
@@ -369,6 +379,70 @@ public int compare(Size lhs, Size rhs) {
         }
     }
 
+    @Override
+    public void takePicture() {
+        imageReader = ImageReader.newInstance(previewSize.getWidth(), previewSize.getHeight(), ImageFormat.JPEG, 1);
```

Comment:
We should not use the preview size here. We could reuse how we choose the video size.